docs/mp/introduction.adoc

```diff
@@ -0,0 +1,152 @@
+
+| Jakarta JSON Processing (JSONP)
+| link:{jakarta-jsonp-spec-url}[{version-lib-jakarta-jsonp-api}]
+| API to parse, generate, transform, and query JSON docs
+
+| Jakarta JSON Binding (JSONB)
+| link:{jakarta-jsonb-spec-url}[{version-lib-jakarta-jsonb-api}]
+| Binding framework for converting POJOs to/from JSON docs
+
+| xref:{rootdir}/mp/jaxrs/server-configuration.adoc[Jakarta RESTful Web Services (JAX-RS)]
+| link:{jakarta-jaxrs-spec-url}[{version-lib-jakarta-jaxrs-api}]
+| API to develop web services following the REST pattern
+
+| xref:{rootdir}/mp/persistence.adoc#JPA[Jakarta Persistence (JPA)]
+| link:{jakarta-persistence-spec-url}[{version-lib-jakarta-persistence-api}]
+| Management of persistence and object/relational mapping
+
+| xref:{rootdir}/mp/persistence.adoc#JTA[Jakarta Transactions (JTA)]
+| link:{jakarta-transactions-spec-url}[{version-lib-jakarta-transaction-api}]
+| Allows handling transactions consistent with X/Open XA-spec
+
+| xref:{rootdir}/mp/websocket.adoc[Jakarta WebSocket]
+| link:{jakarta-websocket-spec-url}[{version-lib-jakarta-websockets-api}]
+| API for Server and Client Endpoints for WebSocket protocol
+
+|===
+
+== Supported MicroProfile Specifications
+
+|===
+| Specification | Version | Description
+
+| xref:{rootdir}/mp/config/introduction.adoc[MicroProfile Config]
+| link:{microprofile-config-spec-url}[{version-lib-microprofile-config}]
+| A flexible configuration framework with support for multiple sources and
+ formats
+
+| xref:{rootdir}/mp/fault-tolerance.adoc[MicroProfile Fault Tolerance]
+| link:{microprofile-fault-tolerance-spec-url}[{version-lib-microprofile-fault-tolerance-api}]
+| Common strategies for various system problems such as time-outs, retries, Circuit Breaker, etc.
+
+| xref:{rootdir}/mp/graphql.adoc[MicroProfile GraphQL]
+| link:{microprofile-graphql-spec-url}[{version-lib-microprofile-graphql}]
+| API for working with GraphQL
+
+| xref:{rootdir}/mp/health.adoc[MicroProfile Health]
+| link:{microprofile-health-spec-url}[{version-lib-microprofile-health}]
+| Health checks for automatic service restart/shutdown
+
+| xref:{rootdir}/mp/jwt.adoc[MicroProfile JWT Auth]
+| link:{microprofile-jwt-spec-url}[{version-lib-microprofile-jwt}]
+| Defines a compact and self-contained way for securely transmitting information between parties as a JSON object
+
+| xref:{rootdir}/mp/lra.adoc[MicroProfile Long-Running Actions (LRA)]
+| link:{microprofile-lra-spec-url}[{version-lib-microprofile-lra-api}]
+| Distributed transactions for microservices following SAGA pattern
+
+| xref:{rootdir}/mp/metrics/metrics.adoc[MicroProfile Metrics]
+| link:{microprofile-metrics-spec-url}[{version-lib-microprofile-metrics-api}]
+| Telemetry data in Prometheus format
+
+| xref:{rootdir}/mp/openapi.adoc[MicroProfile Open API]
+| link:{microprofile-reactive-messaging-spec-url}[{version-lib-microprofile-reactive-messaging-api}]
+| Tracking service calls in distributed environment
```

Comment:
Open API is documentation of microservice endpoints

Reply:
Fixed